pkg/controller/resourceclaim/controller_test.go

```diff
@@ -603,7 +603,7 @@ func makeGeneratedClaim(podClaimName, generateName, namespace, classname string,
 			Name:         fmt.Sprintf("%s-%d", generateName, createCounter),
 			GenerateName: generateName,
 			Namespace:    namespace,
-			Annotations:  map[string]string{"resource.kubernetes.io/pod-claim-name": podClaimName},
+			Annotations:  map[string]string{podResourceClaimAnnotation: podClaimName},
```

Comment:
That the test uses the constant was intentionally: changing the value of that constant could break upgrade/downgrade scenarios, so changing the constant should show up as a unit test failure.

Reply:
got it, thanks for the reminder